test/elixir_auth_google_test.exs

```diff
@@ -30,6 +30,15 @@ defmodule ElixirAuthGoogleTest do
     assert ElixirAuthGoogle.generate_oauth_url(conn) =~ "https://accounts.google.com/o/oauth2/v2/auth?response_type=code"
   end
 
+  test "get Google login url with state" do
+    Application.put_env(:elixir_auth_google, :google_client_id, 123)
```

Comment:
@SimonLab are we using this?

Reply:
Good catch, we indeed don't use it anymore. I'm removing this now.

Reply:
Cool. thanks for confirming. 👍